src/stretchy.js

```diff
@@ -47,6 +47,9 @@ const stretchyCodePoint: {[string]: string} = {
     xtwoheadrightarrow: "\u21a0",
     xlongequal: "=",
     xtofrom: "\u21c4",
+    xrightleftarrows: "\u21c4",
+    xrightequilibrium: "\u21cc",  // Not a perfect match.
+    xleftequilibrium: "\u21cb",   // None better available.
```

Comment:
It took me a couple of times to realize what the difference between `u21cc` and `u21cb`.